packages/node_modules/@ciscospark/widget-message-meet/src/container.js

```diff
@@ -20,7 +20,7 @@ import Button from '@ciscospark/react-component-button';
 import Timer from '@ciscospark/react-component-timer';
 import {ICON_TYPE_WAFFLE} from '@ciscospark/react-component-icon';
 import {metrics} from '@ciscospark/react-component-utils';
-import {MercuryComponent} from '@ciscospark/redux-module-mercury';
+import {connectToMercury} from '@ciscospark/redux-module-mercury';
```

Comment:
Heh... although after I took it out, I realized the Component did make things succinct. Still haven't come up with a great way to manage these connector pieces besides creating action thunks for them. Any thoughts?